css/custom.css

```diff
@@ -133,6 +157,108 @@ img{
   border-top: 3px solid #000000;
 }
 
+/* Sharing Buttons for Social Media */
+.box-btn-media{
+  position: fixed;
+  top: 50%;
+  -webkit-transform: translateY(-50%);
+  -ms-transform: translateY(-50%);
+  transform: translateY(-50%);
+}
+
+.box-btn-media,
+.box-btn-media ul,
+.box-btn-media li {
+  margin: 0;
+  padding: 0;
+  width: fit-content;
+  height: fit-content;
+  list-style: none;
+}
+
+ul li:first-of-type .sm-btn {
+  border-top-right-radius : 5px;
+}
+
+ul li:last-of-type .sm-btn {
+  border-bottom-right-radius : 5px;
+}
+
+.sm-link,
```

Comment:
Do we need to create all these classes?
Can we not just use the bootstrap helper classes?